### ISSUE TITLE
review 'attention deficit-hyperactivity disorder'

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#1930]
related to:
https://github.com/monarch-initiative/mondo/pull/1911 and #1764 

There is a comment on MONDO_0007743 'attention deficit-hyperactivity disorder' to merge with MONDO_0005302 'attention deficit disorder'

These are separate classes in NCIT:

![image](https://user-images.githubusercontent.com/50745/91761537-ee57cd80-eb89-11ea-9e75-61e40b6912fc.png)

From @cmungall:

Also, always pay close attention to OMIM equivs in a merge. 

OMIM has an unsplit entry
https://www.omim.org/entry/143465

And susceptibility entries
https://www.omim.org/entry/608903

It's not clear it's corrrect to equiv the OMIM unsplit entry to a common disorder

I think we may need a separate entry for the susceptibility class

Note - we have a separate class for the susceptibility class:
MONDO_0012149 'attention deficit-hyperactivity disorder, susceptibility to, 1'

**Comment by nicolevasilevsky** [comment]
open ticket with NCIT
- should ADD and ADHD be siblings?

**Comment by nicolevasilevsky** [comment]
From NCIT:

Though ADD and ADHD are often equated , ADD is still used in particular to describe the inattentive/distractible subtype of ADHD. The terms have evolved such that what was once primarily called ADD was changed to ADHD as subsequent changes were made to the DSM (https://add.org/adhd-facts/). The change was spurred by hyperactivity/impulsivity being noted as a key characteristic especially among children.  The current DSM-5 (https://www.aafp.org/dam/AAFP/documents/patient_care/adhd_toolkit/adhd19-assessment-table1.pdf) recognizes the subtypes (presentations) of ADHD.

We have renamed Attention Deficit Disorder (C35092) with a PT of "Attention Deficit Hyperactivity Disorder, Inattentive Type" and kept "Attention Deficit Disorder" and "ADD" as synonyms. It is now a child of Attention Deficit Hyperactivity Disorder (C97160) since it's considered a subtype.  Lastly, we updated the definition of C97160.
